sonar/common/jsonschemas/masked-v1.0.0.json

```diff
@@ -0,0 +1,30 @@
+{
+  "title": "Visibility",
+  "type": "string",
+  "enum": [
+    "not_masked",
+    "masked_for_all",
+    "masked_for_external_ips"
+  ],
+  "description": "A masked document is visible in the professional interface, but not in the public interface.",
+  "default": "not_masked",
+  "form": {
+    "options": [
+      {
+        "label": "Public",
+        "value": "not_masked"
+      },
+      {
+        "label": "Private",
+        "value": "masked_for_all"
+      },
+      {
+        "label": "Restricted to the organisation",
+        "value": "masked_for_external_ips"
+      }
+    ],
```

Comment:
Why not:

```suggestion
    "options": [
      {
        "label": "Public",
        "value": "public"
      },
      {
        "label": "Private",
        "value": "private"
      },
      {
        "label": "Restricted to the organisation",
        "value": "restricted_to_organisation"
      }
    ],
```
?

Reply:
Theses values are already in the document, changing this needs a data correction.